public/index.ts

```diff
@@ -11,12 +12,14 @@ import {
   RENDERMODE_RAYMARCH,
   SKY_LIGHT,
 } from "../src";
+import { State } from "./types";
+import { defaultImageInfo } from "../src/Volume";
 
-let view3D = null;
+let view3D: View3d;
 
-const myState = {
+const myState: State = {
```

Comment:
I thought about renaming `myState` as `state` but it was going to be way too many more changes.